level_maker.cpp

```diff
@@ -2287,6 +2297,9 @@ PLevelMaker LevelMaker::topLevel(RandomGen& random, optional<CreatureFactory> fo
       case SettlementType::CAVE:
         queue = dragonCaveMaker(settlement);
         break;
+      case SettlementType::HUGE_CAVERN:
+        queue = hugeCavernMaker(settlement);
```

Comment:
The hugeCavenMaker() does something weird, it places stone floors outside, even under trees. Just use dragonCaveMaker() here. We can change the floors later.